src/encoded/ingestion/table_utils.py

```diff
@@ -0,0 +1,787 @@
+            table contain this information. Version and Date are in the second column
+            while fields are across the third row, as below:
+                ,version=v1.5, ...
+                ,date=12/1/2019, ...
+                field1, field2, field3, ...
+
+        Returns:
+            3 tuple - version, date, fields
+        """
+        version, date, fields = None, None, None
+        with open(self.mapping_table, 'r', encoding='utf-8-sig') as f:
+            reader = csv.reader(f)
+            for row_idx, row in enumerate(reader):
+                if row_idx == 0:
+                    version = row[1].split('=')[1].strip()
+                elif row_idx == 1:
+                    date = row[1].split('=')[1].strip()
+                elif row_idx == 2:
+                    fields = self.process_fields(row)
+                else:
+                    break # we are done with this step
+        logger.info('Mapping table Version: %s, Date: %s\n' % (version, date))
+        logger.info('Mapping table fields: %s\n' % (", ".join(fields)))
+        return version, date, fields
+
+    def process_annotation_field_inserts(self):
+        """ Processes the annotation fields in the mapping table to produce inserts
+            Note that project and institution are required fields on the annotation
+            field schema and are not set here
+
+        :returns: list of annotation field inserts
+        """
+        inserts = []
+        with open(self.mapping_table, 'r', encoding='utf-8-sig') as f:
+            reader = csv.reader(f)
+            for row_idx, row in enumerate(reader):
+                insert = {}
+                if row_idx <= MappingTableHeader.HEADER_ROW_INDEX:  # skip header rows
+                    continue
+                for field_name, entry in zip(self.fields, row):
+                    if field_name not in self.annotation_field_schema['properties'] or not entry:
+                        continue
+                    if field_name in MappingTableHeader.INTEGER_FIELDS:  # handle int fields
+                        if entry is not None:  # entry=0 is a normal value
+                            insert[field_name] = int(entry)
+                    elif field_name in MappingTableHeader.BOOLEAN_FIELDS:  # handle bool fields
+                        if entry is not None:
+                            if entry == 'Y':
+                                insert[field_name] = True
+                            else:  # assume False if anything other than 'Y' is present
+                                insert[field_name] = False
+                    elif field_name in ['enum_list']:  # handle enum fields
+                        if entry is not None:
+                            field_type = row[self.FIELD_TYPE_INDEX]
+                            val_list = []
+                            if field_type == 'string':
+                                val_list = [en.strip() for en in entry.split(',') if en.strip()]
```

Comment:
I _think_ (untested) it will work to do this, so you don't have to do the strip twice:
```suggestion
                                val_list = [en for en in map(str.strip, entry.split(',')) if en]
```
Although you could also factor this out into a separate function since you seem to do it a lot. It would be a fine addition to `misc_utils`:
```
def parse_string_list(string_list):
    """
    Given a comma-separated list as a string, returns the list of strings it represents, 
    trimming whitespace around entries and ignoring empty entries.
    Examples:
        >>> parse_string_list("a,b,c")
        ['a', 'b', 'c']
        >>> parse_string_list("a,bar,, alpha omega , some.thing  ")
        ['a', 'bar', 'alpha omega', 'some.thing']
    """
    return [s for s in map(str.strip, string_list.split(',')) if s]